tapgarden/batch.go

```diff
@@ -76,6 +76,62 @@ type MintingBatch struct {
 	taprootAssetScriptRoot []byte
 }
 
+type VerboseBatch struct {
+	// CreationTime is the time that this batch was created.
+	CreationTime time.Time
+
+	// HeightHint is the recorded block height at time of creating this
+	// batch. We use it to know where to start looking for the signed batch
+	// transaction.
+	HeightHint uint32
+
+	// batchState is the state of the batch.
+	batchState atomic.Uint32
+
+	// BatchKey is the unique identifier for a batch.
+	BatchKey keychain.KeyDescriptor
+
+	// Seedlings is the set of unsealed seedlings for this batch. This maps
+	// an asset's name to the seedling itself.
+	//
+	// NOTE: This field is only set if the state is BatchStateFrozen or
+	// BatchStatePending.
+	Seedlings map[string]*UnsealedSeedling
```

Comment:
Should we perhaps just embed the `MintingBatch` struct and call this `UnsealedSeedlings` to avoid the code duplication? Because every other field is identical to the normal batch.
Or we could even go with `Seedlings fn.Either[map[string]*Seedling, map[string]*UnsealedSeedling]`. Or extend the `Seedling` itself to have a boolean `sealed` and an additional field `*PendingAssetGroup`?